lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -532,11 +532,18 @@ lldb::VariableSP SwiftExpressionParser::FindSelfVariable(Block *block) {
   return variable_list_sp->FindVariable(ConstString("self"));
 }
 
-static void AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
-                               SwiftASTContextForExpressions &swift_ast_context,
-                               SwiftASTManipulator &manipulator,
-                               lldb::DynamicValueType use_dynamic,
-                               lldb::BindGenericTypes bind_generic_types) {
+/// Adds the type aliases the type-checker needs to type-check the expression.
+///
+///
```

Comment:
**Cosmetic suggestion**

As pretty as this looks, we only need one empty `///` line to separate a symbol's abstract from its overview/discussion.

```suggestion
```